packages/grafana-ui/src/components/Slider/_Slider.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../node_modules/rc-slider/assets/index.css';
+
+.rc-slider-tooltip {
```

Comment:
It would be nice to have these overrides in emotion so we can be consistant with other components. See TimePickerCalendar for an example.

Reply:
That is a good comment! Honestly, I am not sure which one is preferred, because I've seen examples of both and I felt that this would be more preferred way, as we still have to create _Slider.scss to import css from _node_modules/rc-slider/assets/index.css_, so we just can use it to store overrides as well. But if it is not, I can definitely move overrides to emotion.

Reply:
Also, when I looked at the components, that have css file, the overrides are usually part of that file.

Reply:
Yeah, we talked about this in our daily and we concluded that we can keep the import in the scss file but move the overrides to the  component (in emotion)

Reply:
Also talked about adding a css loader to our webpack config, then we can import the css file from code as well so no need to have a scss file